src/fa/game_session.py

```diff
@@ -177,13 +187,45 @@ def _turn_state_changed(self, val):
 
     def _launched(self):
         self._logger.info("Game has started")
+        self._add_to_full_gamelog()
+
+    def _limit_replayid_gamelogs(self):
+        nb_replayid_files=0
+        list_game_replayid_files=os.listdir(util.LOG_DIR)
+        list_game_replayid_files.sort()
```

Comment:
This is not an ideal solution, since we could (in theory) play a game with a later uid, then one with an earlier one (a short ladder game, then a custom game, for example). We could sort by modification time like:
```
files = os.listdir(util.LOG_DIR)
files = map(lambda f: os.path.join(util.LOG_DIR, f), files)
files= sorted(files, key=os.path.getmtime)
files = list(map(os.path.basename, files))
```

Reply:
Oh, I didn't know of this case of ladder/custom game for uid. Thanks fixed.